utilities/bump_versions.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
```

Comment:
I'm still not convinced that we need the bumping script at all. Find/raplace is easy in the version.txt file.

Reply:
If we don't have a bumping script, then we need to write documentation that explains the right process for bumping versions (i.e. turn this script into human instructions). This could be a pretty error-prone process for newbies; I want the instructions for standard releases to be easy.